stable/sentry/templates/service.yaml

```diff
@@ -7,6 +7,10 @@ metadata:
     chart: "{{ .Chart.Name }}-{{ .Chart.Version | replace "+" "_" }}"
     release: "{{ .Release.Name }}"
     heritage: "{{ .Release.Service }}"
+  annotations:
+    {{- range $key, $value := .Values.service.annotations }}
```

Comment:
would it be cleaner to use toYaml instead of the loop - and at the same time conditionally?
See https://github.com/helm/helm/blob/master/docs/charts_tips_and_tricks.md